ida/tutil.py

```diff
@@ -12,8 +12,7 @@ def _add_type (name, decl, attr, pre, post, mode):
     if mode == ADD_TYPE.ENSURE_UNIQUE:
       raise RuntimeError ("struct %s already exists" % (name))
     elif mode == ADD_TYPE.REPLACE:
-      print ("## removed existing", name)
-      ida_typeinf.del_named_type (cvar.idati, name, ida_typeinf.NTF_TYPE)
+      print ("## replaced existing", name)
```

Comment:
This isn't that much of an issue, is it? You're not supposed to run it a hundred times per idb anyway -- assuming the script isn't the crap it is. Was this a real-world issue or a result of debugging?

Reply:
via Discord: Reason to remove the `del_named_type` was that it left lots of deleted local type definitions.

I have no idea why I had seen this as required in the past. Double checked it does indeed replace, even if the struct was modified after importing.